src/app/public/modules/restricted-view/restricted-view-auth.service.ts

```diff
@@ -39,11 +59,27 @@ export class SkyRestrictedViewAuthService  {
           if (typeof permissions === 'number') {
             permissions = [permissions];
           }
+
           if (permissions.indexOf(1) > -1) {
+            this.setHasBeenAuthenticated();
             this.isAuthenticated.next(true);
+            return;
           }
         }
+        // NOTE: Emitting here allows for the watching of when any response is returned.
```

Comment:
In my opinion, we could remove these comments altogether since the source code explains itself (I actually got myself confused because of the comments 😅 ).

Reply:
Done